### PR TITLE
customer service: save new customer

### PR DESCRIPTION
**Pull request by chukwudozie**

### CODE REVIEW

File: src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.Data;
+
+@Data
+public class CreateCustomerDto {
+
+    private String firstName;
+
+    private String lastName;
+
+    private String phoneNumber;
```
Comment (ifeanyichukwuOtiwa-sports) [inline src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java]: the database stores 11 digits ensure that characters are not more than 11.

File: src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package dev.code.digital_lending_microservice.payload.request;
+
+import lombok.Data;
+
+@Data
+public class CreateCustomerDto {
+
+    private String firstName;
```
Comment (ifeanyichukwuOtiwa-sports) [inline src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java]: Ensure that firstname and lastname are not null and not empty

File: src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java (hunk 1 above)
Comment (chukwudozie) [reply src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java]: ok

File: src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java (hunk 2 above)
Comment (chukwudozie) [reply src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java]: ok

**Review by ifeanyichukwuOtiwa-sports — APPROVED** [review]

**Comment by ifeanyichukwuOtiwa-sports** [comment]
clean, LGTM

**Pull request merged**